.github/workflows/ci.yaml

```diff
@@ -0,0 +1,46 @@
+# SPDX-FileCopyrightText: Copyright (C) 2025 Opal Health Informatics Group at the Research Institute of the McGill University Health Centre <john.kildea@mcgill.ca>
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+  workflow_dispatch:
+  merge_group:
```

Comment:
Do we need these empty sections?

Reply:
Yes. Otherwise it won't react to those events.

Reply:
Ah ok, I didn't realize adding them empty means "run on this event". All good.